### PR TITLE
avoid confusion with modern Medtronic pump series

### PR DESCRIPTION
**Pull request by marionbarker**

### CODE REVIEW

File: MinimedKit/PumpManager/MinimedPumpManager.swift [hunk 1]
```diff
@@ -909,7 +909,7 @@ extension MinimedPumpManager {
 // MARK: - PumpManager
 extension MinimedPumpManager: PumpManager {
     
-    public static let localizedTitle = LocalizedString("Minimed 500/700 Series", comment: "Generic title of the minimed pump manager")
+    public static let localizedTitle = LocalizedString("Minimed (some models)", comment: "Generic title of the minimed pump manager")
```
Comment (ps2) [inline MinimedKit/PumpManager/MinimedPumpManager.swift:912]: Would prefer to just remove the 500/700 if we think it's causing confusion.  The "title" is not the place to try to have verbiage describing what it is.  It should just be a name.

Comment (ps2) [reply MinimedKit/PumpManager/MinimedPumpManager.swift:912]: If the confusion about this is very rare, also fine to leave it the way it is; it's generally true.

**Comment by marionbarker** [comment]
Missed the comments. I'll fix it now.

**Comment by marionbarker** [comment]
This is the graphic with commit 41b3169.
Is there something I should do about translations or is that handled differently.

pr-10-minimed-some-models

**Comment by ps2** [comment]
Re translation: The pump manager titles generally represent proper names and should not be translated.

**Pull request merged**